readme.md

```diff
@@ -44,10 +44,11 @@ Heads up for Windows + Vagrant users: Start by applying the fix in [issue #61](h
 ```bash
 composer create-project jadjoubran/laravel5-angular-material-starter --prefer-dist
 cd laravel5-angular-material-starter
-#fix database credentials in .env
+#fix database credentials, add API_API_PREFIX and API_DOMAIN entries to .env
```

Comment:
No worries
Your contribution is greatly appreciated :smile: 
Make sure to checkout the new #105 for Version 3